admin_manual/installation/system_requirements.rst

```diff
@@ -24,7 +24,7 @@ For best performance, stability and functionality we have documented some recomm
 +------------------+-----------------------------------------------------------------------+
 | Database         | - **MySQL 8.0+ or MariaDB 10.2/10.3/10.4/10.5** (recommended)         |
 |                  | - Oracle Database 11g (*only as part of an enterprise subscription*)  |
-|                  | - PostgreSQL 10/11/12/13/14                                           |
+|                  | - PostgreSQL 10/11/12/13/14/15                                           |
```

Comment:
```suggestion
|                  | - PostgreSQL 10/11/12/13/14/15                                        |
```

Reply:
(layout of the markdown table is messed up (you need to remove 3 spaces)

Reply:
done